src/App.js

```diff
@@ -6,13 +6,15 @@ import Homepage from "./pages/HomePage";
 import { HashRouter as Router, Route } from "react-router-dom";
 import PublicWorksLandingPage from "./pages/PublicWorksLandingPage";
 import AdoptablePetsPage from "./pages/AdoptablePetsPage";
+import Images from "./pages/ImagesPage";
 
 function App() {
   return (
     <Router>
       <Route exact path="/" component={Homepage} />
       <Route exact path="/public-works" component={PublicWorksLandingPage} />
       <Route exact path="/adoptable-pets" component={AdoptablePetsPage} />
+      <Route exact path="/images" component={Images} />
```

Comment:
to match the other interal pages can we name this component "ImagesPage" here and on line 9 where we import it?

Reply:
@tmccoy529 ill just do this real quick, since im in here.